### ISSUE TITLE
OUTFIT

### ISSUE THREAD

**Issue by Flordy** [multimokia/MAS-Submod-Auto-Outfit-Change#12]
How do you add outfits? it only refers to auto hair change but never the outfits. Lack of explanation in your project here.

**Comment by digital-silk** [comment]
You're right! It would be much easier if devs would leave us some more understandable instructions. I still have no idea how to make it work...
But as I know, we need to open the .rpy file inside of the submod and somehow edit it... Wish I had something to open it :(

**Comment by multimokia** [comment]
You don't need an rpy, you simply need to modify the json file to add the required `ex_props` field. You can look at other spritepack jsons for examples.

**Comment by bepbopyobop** [comment]
say all of that again but like youre explaining it to a 2 year old
